ui_advanced_config.py

```diff
@@ -320,37 +355,47 @@ def build_ui(self):
             master=self.objectsColumn, command=self.delete_object, text="Delete"
         )
         self.objectsIDLabel = uiLabel(master=self.objectsColumn, text="ID:")
-        self.objectsIDEntry = uiEntry(master=self.objectsColumn)
+        self.objectsIDEntry = EntryHelp(master=self.objectsColumn, text="To be added.")
         self.objectsNameLabel = uiLabel(master=self.objectsColumn, text="Name:")
-        self.objectsNameEntry = uiEntry(master=self.objectsColumn)
+        self.objectsNameEntry = EntryHelp(
+            master=self.objectsColumn, text="To be added."
+        )
         self.objectsFillAliveLabel = uiLabel(
             master=self.objectsColumn, text="Fill Alive:"
         )
-        self.objectsFillAliveEntry = uiEntry(
-            master=self.objectsColumn,
+        self.objectsFillAliveEntry = EntryHelp(
+            master=self.objectsColumn, text="To be added."
         )
         self.objectsFillDeadLabel = uiLabel(
             master=self.objectsColumn, text="Fill Dead:"
         )
-        self.objectsFillDeadEntry = uiEntry(master=self.objectsColumn)
+        self.objectsFillDeadEntry = EntryHelp(
+            master=self.objectsColumn, text="To be added."
+        )
         self.objectsTextLabel = uiLabel(master=self.objectsColumn, text="Text:")
-        self.objectsTextEntry = uiEntry(master=self.objectsColumn)
+        self.objectsTextEntry = EntryHelp(
+            master=self.objectsColumn, text="To be added."
+        )
         self.objectsHealthLabel = uiLabel(master=self.objectsColumn, text="Health:")
-        self.objectsHealthEntry = uiEntry(master=self.objectsColumn)
-        self.objectsHealthEntry.config(
+        self.objectsHealthEntry = EntryHelp(
+            master=self.objectsColumn, text="To be added."
+        )
+        self.objectsHealthEntry.entry.config(
             validate="all", validatecommand=(self.validateNum, "%P")
         )
         self.objectsDensityLabel = uiLabel(master=self.objectsColumn, text="Density:")
-        self.objectsDensityEntry = uiEntry(master=self.objectsColumn)
-        self.objectsDensityEntry.config(
+        self.objectsDensityEntry = EntryHelp(
+            master=self.objectsColumn, text="To be added."
+        )
+        self.objectsDensityEntry.entry.config(
             validate="all", validatecommand=(self.validateNum, "%P")
         )
         self.objectsCompIDsLabel = uiLabel(master=self.objectsColumn, text="Comp IDs:")
         self.objectsCompIDsCombo = uiComboBox(master=self.objectsColumn)
         self.objectsPointsCountLabel = uiLabel(
             master=self.objectsColumn, text="Points Count:"
         )
-        self.objectsPointsCountEntry = uiEntry(master=self.objectsColumn)
+        self.objectsPointsCountCombo = uiComboBox(master=self.objectsColumn)
```

Comment:
comp id's and point count do not have help boxes
<img width="343" alt="Screen Shot 2024-04-02 at 2 53 28 PM" src="https://github.com/Leading-AI-Development-Solutions/MAIA/assets/100381732/4a92eaa8-faca-4bc1-93f3-4d7e9f2f7306">

Reply:
in progress